### ISSUE TITLE
Public datasets list

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/autokeras#1219]
This is a list of public datasets, that we need to ensure AutoKeras is having good performances on.
Welcome to post more.

| Name | API | Code | Metric | Results |
| - | - | - | - | - |
| [MNIST](http://yann.lecun.com/exdb/mnist/)  | ImageClassifier| [code](https://github.com/keras-team/autokeras/blob/master/benchmark/mnist.py) | Accuracy | 99.04% |
| [CIFAR10](https://www.cs.toronto.edu/~kriz/cifar.html)   | ImageClassifier|[code](https://github.com/keras-team/autokeras/blob/master/benchmark/cifar10.py) |Accuracy | 97.10% |
| [CIFAR100](https://www.cs.toronto.edu/~kriz/cifar.html)  | ImageClassifier | | Accuracy |
| [IMDB](https://ai.stanford.edu/~amaas/data/sentiment/)  | TextClassifier |[code](https://github.com/keras-team/autokeras/blob/master/benchmark/imdb.py) | Accuracy | 93.93% |
| [Titanic](https://www.tensorflow.org/datasets/catalog/titanic)  | StructuredDataClassifier |  [code](https://github.com/keras-team/autokeras/blob/master/benchmark/imdb.py)| Accuracy | 82.20% |
| [Wine](https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_wine.html#sklearn.datasets.load_wine)  | StructuredDataClassifier | | Accuracy|
| [Iris](https://archive.ics.uci.edu/ml/datasets/iris) | StructuredDataClassifier | | Accuracy |
| [California Housing](https://scikit-learn.org/stable/datasets/index.html#california-housing-dataset)  | StructuredDataRegression |  [code](https://github.com/keras-team/autokeras/blob/master/benchmark/housing.py) | MSE | 0.23
| [Houses](https://github.com/emanhamed/Houses-dataset)  | IO API | | MSE| |

**Comment by mandalbiswadip** [comment]
@haifeng-jin Can I take up few of the datasets from the list and work on them? Let me know which ones are yet to be tested out.

**Comment by haifeng-jin** [comment]
@mandalbiswadip Would you help us test the Wine dataset? Thank you!

**Comment by toohsk** [comment]
@haifeng-jin 
As you commented in the previous issue #1283, I created a pull request.
Would you review and merge it?
Thanks.

**Comment by toohsk** [comment]
Can I try [`Reuters` dataset](https://keras.io/ja/datasets/#_5)?
Or are there any prioritized dataset? @haifeng-jin

**Comment by haifeng-jin** [comment]
@toohsk Yes, please. Thank you! I think it is good to try these Keras dataset first.
